src/ui/suspense/frameCache.ts

```diff
@@ -123,7 +123,8 @@ export const formattedPointStackCache: Cache<
     const formattedFrames = pointStack.map(frame => {
       updateMappedLocationForPointStackFrame(sourcesById, frame);
       const functionLocation = getPreferredLocation(sourcesById, [], frame.functionLocation);
-      const executionLocation = getPreferredLocation(sourcesById, [], frame.point?.frame ?? []);
```

Comment:
`?? []` couldn't work correctly here before because that would just always lead to a failed assert in `getPreferredLocation`